docs/self-managed/zeebe-deployment/security/secure-client-communication.md

```diff
@@ -113,12 +113,16 @@ Then start up your gateway with the certificate and key specified above. For exa
 docker run -p 26500:26500 -e ZEEBE_BROKER_NETWORK_HOST=0.0.0.0 -e ZEEBE_BROKER_GATEWAY_SECURITY_ENABLED=true -e ZEEBE_BROKER_GATEWAY_SECURITY_CERTIFICATECHAINPATH=/usr/local/zeebe/cert.pem -e ZEEBE_BROKER_GATEWAY_SECURITY_PRIVATEKEYPATH=/usr/local/zeebe/key.pem --mount type=bind,source="$(pwd)"/cert.pem,target=/usr/local/zeebe/cert.pem --mount type=bind,source="$(pwd)"/key.pem,target=/usr/local/zeebe/key.pem camunda/zeebe
 ```
 
-There is one caveat: in order for the client to accept this self-signed certificate, you will need to trust it. The simplest way is to specify it as part of the client's configuration. For example, if you're using `zbctl`, you can then do `zbctl --certPath cert.pem status`. Refer to the documentation above on how to configure your clients.
+There is one caveat: in order for the client to accept this self-signed certificate, you will need to trust it. The simplest way is to specify it as part of the client's configuration. Refer to the documentation above on how to configure your clients.
 
 ## Troubleshooting authentication issues
 
 Here we will describe a few ways the clients and gateway could be misconfigured and what those errors look like. Hopefully, this will help you recognize these situations and provide an easy fix.
 
+:::note
```

Comment:
I don't love this, but I'm not sure what else to do. The whole troubleshooting section on this page includes zbctl specific instructions.

Reply:
Yeah, it's about troubleshooting failing `zbctl` commands. We could look into what cURL commands you'd run instead and if they could suffer from the same issues. But that will be a HUGE effort for sure 😅

Reply:
Is it on us to maintain a list of zbctl troubleshooting steps? Should the whole section move to zbctl's repo instead?

Reply:
I like the idea of moving it!

Reply:
I'll move this in a separate PR so we can keep this in a good state to get it merged.